Bukkit/src/main/java/com/plotsquared/bukkit/queue/BukkitQueueCoordinator.java

```diff
@@ -271,4 +292,102 @@ private void setWorldBlock(int x, int y, int z, @NonNull BaseBlock block, @NonNu
         return bukkitWorld;
     }
 
+    private boolean isEdge(int layer, int x, int y, int z, BlockVector2 blockVector2, LocalChunk localChunk) {
+        if (layer == 0 || layer == localChunk.getBaseblocks().length - 1) {
+            return false;
+        }
+        if (x == 0) {
+            LocalChunk localChunkX = getBlockChunks().get(blockVector2.withX(blockVector2.getX() - 1));
+            if (localChunkX == null || localChunkX.getBaseblocks()[layer] == null || localChunkX.getBaseblocks()[layer][ChunkUtil.getJ(
```

Comment:
This formatting is really odd. Can you wrap on the `||` instead?

Reply:
It's the editorconfig 🙃

Reply:
Manually doing the line break should work and is a lot more readable

Reply:
It'll get undone when someone updates code reformats lol